supabase/functions/gh-webhook/index.ts

```diff
@@ -0,0 +1,50 @@
+// Follow this setup guide to integrate the Deno language server with your editor:
+// https://deno.land/manual/getting_started/setup_your_environment
+// This enables autocomplete, go to definition, etc.
+
+import { serve } from "https://deno.land/std@0.131.0/http/server.ts"
+import { supabaseClient } from "../_shared/supabaseClient.ts";
+
+console.log("GitHub webhooks functions started.")
+
+serve(async (req) => {
+  const { action, ...body } = await req.json()
+
+  const package_url: string = body.package.package_version.package_url;
+  const [image_name, image_tag] = package_url.split(':');
+  const { data: connector, error }: { data: { id: string, image_name: string } | null, error: any } = await supabaseClient
+    .from("connectors")
+    .select("id,image_name")
+    .eq("image_name", image_name)
+    .single();
+
+  if (error != null) {
```

Comment:
I would expect a return or throw here, otherwise we're going to continue on to the update request with a null/undefined `connector`, and try to do a `update connector_tags where connector_id = null/undefined` which seems not great

Reply:
Good point thank you!